config/drupal-vm.settings.php

```diff
@@ -31,6 +31,8 @@
 // Production template is 'arn:aws:sns:us-east-1:512686554592:bus-%s--dev'.
 $settings['jcms_sns_topic_template'] = 'arn:aws:sns:local:000000000000:%s';
 $settings['jcms_sqs_region'] = 'us-east-1';
-$settings['jcms_articles_endpoint'] = 'https://prod--gateway.elifesciences.org/articles/%s/versions';
-$settings['jcms_all_articles_endpoint'] = 'https://prod--gateway.elifesciences.org/articles';
+$settings['jcms_gateway'] = 'https://prod--gateway.elifesciences.org';
+$settings['jcms_all_articles_endpoint'] = $settings['jcms_gateway'] . '/articles';
+$settings['jcms_articles_endpoint'] = $settings['jcms_gateway'] . '/articles/%s/versions';
+$settings['jcms_article_fragment_images_endpoint'] = $settings['jcms_gateway'] . '/articles/%s/fragment/image';
```

Comment:
we need to add `jcms_article_fragment_images_endpoint` in the settings provided by the formula, also `jcms_gateway`?